Makefile

```diff
@@ -99,7 +102,7 @@ vet:
 
 staticcheck: $(STATICCHECK)
 	@echo ">> running staticcheck"
-	@$(STATICCHECK) $(pkgs)
+	@$(STATICCHECK) -ignore "$(STATICCHECK_IGNORE)" $(pkgs)
```

Comment:
Please don't introduce staticcheck violations.

Reply:
This was discussed on IRC with @beorn7 and @SuperQ 
Without this we will get:
```
node_exporter.go:86:32: prometheus.InstrumentHandlerFunc is deprecated: InstrumentHandlerFunc is deprecated for the same reasons as InstrumentHandler is. Use the tooling provided in package promhttp instead.  (SA1019)
```
Another solution is to use `prometheus@v0.8.0` instead of `v0.9.0-pre1`

Reply:
Ok. Please add a comment mentioning InstrumentHandlerFunc and when that ignoring can be removed again.

Reply:
Apparently this will also warn in client_golang v0.8.0 as well. :(